src/common/util.cpp

```diff
@@ -79,3 +76,36 @@ std::vector<uint8_t> hexToBytes(std::string hexStr) {
 
     return bytes;
 }
+
+const int compareFloats(const float a, const float b, const size_t epsilon) {
+    /*
+    A simple decimal comparison might involve subtraction, fabs, and the result being less than a smaller decimal.
+    This method converts the IEEE float values into lexicographically-ordered two's-complement integers.
+    If (float)a > (float)b then (int)a > (int)b; but, precision concerns are lessened.
+    The error becomes a count of "the maximum number of lexico-numbers allowed between equal lexico-a and lexico-b."
+    To avoid encountering a NaN value, error can not risk being too big, however.
+    */
+    if(epsilon >= 4194304) {
```

Comment:
Read your other comment - seems to make sense to me (somewhat :) ). No problem if there isn't some relatively easy meaning for this, as it would be somewhat of an insane epsilon to use in our cases anyway (as you noted). I think it's totally fine to leave as-is, was just wondering if there was some quick explanation.